debian.aws/config/OVERRIDES

```diff
@@ -5,14 +5,17 @@ CONFIG_6LOWPAN=n
 CONFIG_6LOWPAN_NHC=n
 CONFIG_BT=n
 CONFIG_CAIF=n
+CONFIG_CFG80211=n
 CONFIG_HAMRADIO=n
 CONFIG_IEEE802154=n
+CONFIG_LIB80211=n
 CONFIG_NET_9P=n
 CONFIG_NET_DSA=n
 CONFIG_NFC=n
 CONFIG_RFKILL=n
 CONFIG_WIMAX=n
 CONFIG_WIRELESS=n
+CONFIG_WLAN=n
```

Comment:
I think this one belongs under the `drivers` section below because it results in the omission of all WLAN drivers under the `drivers` directory.

Reply:
Sure.